astropy/units/quantity_helper/function_helpers.py

```diff
@@ -1042,3 +1055,54 @@ def eig(a, *args, **kwargs):
     from astropy.units import dimensionless_unscaled
 
     return (a.value,)+args, kwargs, (a.unit, dimensionless_unscaled), None
+
+
+@function_helper(module=np.lib.recfunctions)
+def structured_to_unstructured(arr, *args, **kwargs):
+    """
+    Convert a structured quantity to an unstructured one.
+    This only works if all the units are compatible.
+    """
+    from astropy.units import StructuredUnit
+
+    target_unit = arr.unit.values()[0]
+
+    def replace_unit(x):
+        if isinstance(x, StructuredUnit):
+            return x._recursively_apply(replace_unit)
+        else:
+            return target_unit
+
+    to_unit = arr.unit._recursively_apply(replace_unit)
+    return (arr.to_value(to_unit), ) + args, kwargs, target_unit, None
+
+
+def _build_structured_unit(dtype, unit):
+    """Build structured unit from dtype
+
+    Parameters
+    ----------
+    dtype : `numpy.dtype`
+    unit : `astropy.units.Unit`
+
+    Returns
+    -------
+    `astropy.units.Unit` or tuple
+    """
+    if dtype.fields is None:
+        return unit
+
+    return tuple(_build_structured_unit(v[0], unit) for v in dtype.fields.values())
+
+
+@function_helper(module=np.lib.recfunctions)
+def unstructured_to_structured(arr, dtype, *args, **kwargs):
+    """
+    Convert a structured quantity to an unstructured one.
+    This only works if all the units are compatible.
```

Comment:
This works always, correct? Given that, I'd just remove the docstring, so that when I get around to making a nice sphinx page of the overrides (like https://docs.astropy.org/en/latest/utils/masked/index.html#module-astropy.utils.masked.function_helpers for `Masked), it will only list those that are not obvious.

Reply:
Yes, I believe it should always work. Removing now.

Reply:
✅